backend/api/tests/unit/get-supabase-token.unit.test.ts

```diff
@@ -0,0 +1,46 @@
+jest.mock('jsonwebtoken');
+
+import { getSupabaseToken } from "api/get-supabase-token";
+import * as jsonWebtokenModules from "jsonwebtoken";
+import * as constants from "common/envs/constants";
+import { AuthedUser } from "api/helpers/endpoint";
+
+describe.skip('getSupabaseToken', () => {
```

Comment:
Change to something like that:
```suggestion
describe('getSupabaseToken', () => {
  it('skip', async () => {})
```

To fix:
```

FAIL tests/unit/get-supabase-token.unit.test.ts
  ● Test suite failed to run

    Your test suite must contain at least one test.
```

After that, all the tests should pass.